css/css-transitions/before-load-001.html

```diff
@@ -1,150 +1,46 @@
-                        'values': {
-                            // run actual test, assertions can be used here!
-                            start: function(test, data, options) {
-                                // identify initial and target values
-                                generalParallelTest.getStyle(data);
-                                // make sure values differ, if they don't, the property could most likely not be parsed
-                                assert_not_equals(data.transition.from, data.transition.to, "initial and target values may not match");
-                                // kick off the transition
-                                generalParallelTest.startTransition(data);
-
-                                // make sure we didn't get the target value immediately.
-                                // If we did, there wouldn't be a transition!
-                                var current = data.transition.computedStyle(data.property);
-                                assert_not_equals(current, data.transition.to, "must not be target value after start");
-                            },
-                            done: function(test, data, options) {
-                                // make sure the property's value were neither initial nor target while transitioning
-                                test.step(generalParallelTest.assertIntermediateValuesFunc(data, 'transition'));
-                            }
-                        },
-                        // test TransitionEnd events
-                        'events': {
-                            done: function(test, data, options) {
-                                // make sure there were no events on parent
-                                test.step(generalParallelTest.assertExpectedEventsFunc(data, 'container', ""));
-                                // make sure we got the event for the tested property only
-                                test.step(generalParallelTest.assertExpectedEventsFunc(data, 'transition', addVendorPrefix(data.property) + ":" + duration));
-
-                                test.step(function() {
-                                    assert_false(isLoad, "load may not have happened yet")
-                                });
-                            }
-                        }
-                    },
-                    // called once all tests are done
-                    done: generalParallelTest.done
-                });
-            }, false);
-        </script>
-        <img src="/delay/?type=gif&amp;delay=3000" alt="dummy image">
-    </body>
+<head>
+<meta charset="utf-8">
+<title>CSS Transitions Test: Transitioning before load event</title>
+<meta name="assert" content="Test checks that transitions are run even before the load event fires">
+<!-- NOTE: This test covers unspecified behavior and should probably be
+     removed. It this behavior *were* specified, it would probably be
+     specified here: -->
```

Comment:
Is this note still current?

Reply:
Yeah, I don't think anything requires this to happen. That said, it passed on the browsers I tested it on. I'm not sure if it makes sense to specify that or not though so perhaps we should just drop the test/